bin/make-base-vm

```diff
@@ -153,6 +159,32 @@ fi
 # Remove cron to work around vmbuilder issue when umounting /dev on target
 removepkg=cron
 
+if [ $DOCKER = "1" ]; then
+
+  addpkg=`echo $addpkg | tr ',' ' '`
+
+  mkdir -p docker
+  cd docker
+
+  # Generate the dockerfile
+  cat << EOF > $OUT.Dockerfile
+FROM $DISTRO:$SUITE
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get --no-install-recommends -y install $addpkg
```

Comment:
Does it make sense to run a `dist-upgrade` here as well? This seems to be done later by gitian-builder nevertheless, but doing it here as well would speed up the later upgrade.

Reply:
I think we should keep the base vm image as close to what lxc and kvm modes would generate. AFAICT, they don't do dist-upgrade either.